qcut/.claude/skills/qcut-api/REF-MEDIA-TIMELINE.md

```
@@ -0,0 +1,220 @@
+interface ClaudeSplitResponse { secondElementId: string | null }
+interface ClaudeMoveRequest { toTrackId: string; newStartTime?: number }
+interface ClaudeSelectionItem { trackId: string; elementId: string }
+
+type ClaudeArrangeMode = "sequential" | "spaced" | "manual";
+interface ClaudeArrangeRequest { trackId: string; mode: ClaudeArrangeMode; gap?: number; order?: string[]; startOffset?: number }
+interface ClaudeArrangeResponse { arranged: { elementId: string; newStartTime: number }[] }
+```
+
+## Media Endpoints
+
+#### List media files
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)